shared/src/main/kotlin/com/egm/stellio/shared/util/DateUtils.kt

```diff
@@ -6,6 +6,8 @@ import java.time.format.DateTimeFormatter
 
 val formatter: DateTimeFormatter = DateTimeFormatter.ISO_INSTANT
 
+const val TEMPLATE: String = "\"YYYY-MM-DD\\\"T\\\"HH24:MI:SS.US\\\"Z\\\"\""
```

Comment:
very specific to PG, better put in in `DBUtils`. With a more meaningful name (e.g., `DATETIME_TEMPLATE`)